plugins/parsers/registry_test.go

```diff
@@ -45,7 +45,7 @@ func TestRegistry_BackwardCompatibility(t *testing.T) {
 	}
 
 	// Define parsers that do not have an old-school init
-	newStyleOnly := []string{"binary", "avro"}
+	newStyleOnly := []string{"binary", "avro", "opentsdb"}
```

Comment:
I removed the `NewOpenTSDBParser` function and am assuming that it's fine for newly added plugins to just be listed here, but it's also fine if I should include support for "old-school" init in the plugin. Just let me know if that's the case and I can add it.